src/app/event/[id]/EventDetail.tsx

```diff
@@ -1,6 +1,5 @@
 'use client';
 
-import format from 'date-fns/format';
```

Comment:
We could write out what bugs the PR is fixing, at least the most major one, or the general category. Don't make the commit 200 characters obvi, but I do believe there si some better alternative than `some`